packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 * Updates Android compileSdkVersion to 31.
 * Fix iOS RunnerUITests search paths.
 
+## 0.8.4+5
```

Comment:
Seems like a rebase mistake, this needs to be at the very beginning of this file.